lib/Templates.jsx

```diff
@@ -119,22 +121,18 @@ export default (function () {
          * @return {String}
          */
         get(templatePath, data = {}) {
```

Comment:
Why was the `try catch` removed?

Reply:
Eslint rule was complaining, try catch that always throws is unnecessary.